python/GangaLHCb/Lib/GaudiRunApp/GaudiRunApp.py

```diff
@@ -0,0 +1,248 @@
+
+from Ganga.GPIDev.Adapters.IPrepareApp import IPrepareApp
+from Ganga.GPIDev.Adapters.IRuntimeHandler import IRuntimeHandler
+from Ganga.GPIDev.Schema import Schema, Version, SimpleItem, GangaFileItem
+
+from Ganga.Utility.Config import getConfig
+
+from Ganga.GPIDev.Lib.File.File import File, ShareDir
+from Ganga.Core import ApplicationConfigurationError, ApplicationPrepareError, GangaException
+
+from Ganga.Utility.logging import getLogger
+
+from Ganga.GPIDev.Base.Proxy import getName, isType, stripProxy
+
+from os import rename, path
+import shutil
+from Ganga.Utility.files import expandfilename
+
+from Ganga.GPIDev.Lib.File.LocalFile import LocalFile
+from GangaDirac.Lib.Files.DiracFile import DiracFile
+
+from GangaDirac.Lib.Backends.DiracBase import DiracBase
+
+import tempfile
+import time
+import subprocess
+
+logger = getLogger()
+cmake_sandbox_name = 'cmake-input-sandbox.tgz'
+build_target = 'ganga-input-sandbox'
+build_dest = 'input-sandbox.tgz'
+defDir = '$HOME/cmtuser'
```

Comment:
There's no need for these to be module-globals. They'd be better off as class attributes to make their scope clearer.

`cmake_sandbox_name` is only used once on the whole module and needn't be a variable at all.